eng/SharedFramework.External.props

```diff
@@ -68,7 +68,6 @@
 
       If these are needed as direct dependencies, it is okay to change them to ExternalAspNetCoreAppReference and move up into sections above.
     -->
-    <_TransitiveExternalAspNetCoreAppReference Include="Newtonsoft.Json"                                    Version="$(NewtonsoftJsonPackageVersion)" />
```

Comment:
@pranavkm @rynowak Is MVC good with this change? We are making the switch to System.Text.Json in SignalR by default now.

@ryanbrandenburg Do our templates reference Newtonsoft anywhere or did we already remove all that?

Reply:
What does this do?

> Do our templates reference Newtonsoft anywhere or did we already remove all that?

MVC's templates still do. We were waiting for https://github.com/dotnet/corefx/issues/36024 to land since it affects quite a few of our tests

Reply:
This removes Newtonsoft.Json from the shared framework. We can't do that if MVC isn't ready to do it though.

My concern is that this is going to land super hot for preview 5 and I want everybody on the same page. Maybe we should leave Newtonsoft in the shared framework, even though we're changing the default.

Reply:
Ok, if we can land this soon that would be best. We need to try our templates after this change and make sure we haven't broken the immediate experience.